mad-dog/src/index.js

```diff
@@ -153,7 +158,16 @@ async function main () {
             numUsers: DEFAULT_NUM_USERS
           })
       )
-      const tests = [test, ...blacklistTests]
+
+      const signUpReplicaSetTest = makeTest(
+        'signUpReplicaSetTest',
+        replicaSetTest.assignReplicaSetAndSyncOnSignUp,
+        {
+          numCreatorNodes: 3,
+          numUsers: 1
+        }
+      )
+      const tests = [signUpReplicaSetTest]//, test, ...blacklistTests]
```

Comment:
why make a new test for this? this seems like it should go in the core consistency test

Reply:
my bad, I accidentally committed this in solely testing the replica set code. Updated